src/main/charts/confluence/templates/statefulset-synchrony.yaml

```diff
@@ -21,7 +21,7 @@ spec:
         {{- include "synchrony.selectorLabels" . | nindent 8 }}
     spec:
       serviceAccountName: {{ include "confluence.serviceAccountName" . }}
-      terminationGracePeriodSeconds: 1
+      terminationGracePeriodSeconds: {{ .Values.terminationGracePeriodSeconds }}
```

Comment:
Just to let you know - I'm working on a similar PR in #250 and I will likely move this property to different value (because I also want to add shutdown command). I will wait until you merge and then start on top of your changes.

Reply:
:+1: